models/im.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+import "time"
+
+type Room struct {
```

Comment:
Room inside of a file named im is kind of confusing. 

Also room and channel are the same thing.  Internally both are rooms.  Not sure if we need this type on this end,

Reply:
will just change this later maybe.